tests/query/processors/test_mandatory_condition_applier.py

```diff
@@ -0,0 +1,80 @@
+import pytest
+from typing import List
+
+# from snuba.clickhouse.columns import ColumnSet, String
+
+from snuba.datasets.schemas.tables import TableSource
+from snuba.query.logical import Query
+from snuba.query.processors.mandatory_condition_applier import MandatoryConditionApplier
+
+from snuba.request.request_settings import HTTPRequestSettings
+from snuba.datasets.schemas import MandatoryCondition
```

Comment:
please sort the imports.
The guidelines on how to do that is the one in PEP8: https://www.python.org/dev/peps/pep-0008/#imports

Most IDEs would do that for you though, no need to do that manually.